### PR TITLE
Manual updates 20240715

### PR DESCRIPTION
**Pull request by moljac**
### Describe your contribution

Updated

- `com.google.android.gms:playservices-ads` - 22.4.0 -> 22.6.0
- `com.google.android.gms:playservices-ads-base` - 22.4.0 -> 22.6.0
- `com.google.android.gms:playservices-ads-lite` - 22.4.0 -> 22.6.0
- `com.google.android.gms:playservices-auth-blockstore` - 16.2.0 -> 16.3.0
- `com.google.android.gms:playservices-games-v2` - 20.1.0 -> 20.1.2
- `com.google.android.gms:playservices-maps` - 18.2.0 -> 19.0.0
- `com.google.android.gms:playservices-measurement` - 21.4.0 -> 21.5.1
- `com.google.android.gms:playservices-measurement-api` - 21.4.0 -> 21.5.1
- `com.google.android.gms:playservices-measurement-base` - 21.4.0 -> 21.5.1
- `com.google.android.gms:playservices-measurement-impl` - 21.4.0 -> 21.5.1
- `com.google.android.gms:playservices-measurement-sdk` - 21.4.0 -> 21.5.1
- `com.google.android.gms:playservices-measurement-sdk-api` - 21.4.0 -> 21.5.1
- `com.google.android.play:core-common` - 2.0.3 -> 2.0.4
- `com.google.firebase:firebase-ads` - 22.4.0 -> 22.5.0
- `com.google.firebase:firebase-ads-lite` - 22.4.0 -> 22.5.0
- `com.google.firebase:firebase-analytics` - 21.4.0 -> 21.5.1
- `com.google.firebase:firebase-analytics-ktx` - 21.4.0 -> 21.5.1
- `com.google.firebase:firebase-crashlytics` - 19.0.1 -> 19.0.3
- `com.google.firebase:firebase-crashlytics-ktx` - 19.0.1 -> 19.0.3
- `com.google.firebase:firebase-crashlytics-ndk` - 19.0.1 -> 19.0.3
- `com.google.firebase:firebase-sessions` - 2.0.1 -> 2.0.3
- `com.google.protobuf:protobuf-javalite` - 4.27.1 -> 4.27.2
- `io.grpc:grpc-android` - 1.60.2 -> 1.61.1
- `io.grpc:grpc-api` - 1.60.2 -> 1.61.1
- `io.grpc:grpc-context` - 1.60.2 -> 1.61.1
- `io.grpc:grpc-core` - 1.60.2 -> 1.61.1
- `io.grpc:grpc-okhttp` - 1.60.2 -> 1.61.1
- `io.grpc:grpc-protobuf-lite` - 1.60.2 -> 1.61.1
- `io.grpc:grpc-stub` - 1.60.2 -> 1.61.1
- `io.grpc:grpc-util` - 1.60.2 -> 1.61.1
- `org.codehaus.mojo:animal-sniffer-annotations` - 1.23 -> 1.24
- `org.ow2.asm:asm` - 9.6 -> 9.7
- `com.google.ads.interactivemedia.v3:interactivemedia` - 3.30.0 -> 3.31.0
- `androidx.annotation:annotation` - 1.7.1 -> 1.8.0
- `androidx.appcompat:appcompat` - 1.6.1 -> 1.7.0
- `androidx.concurrent:concurrent-futures` - 1.1.0 -> 1.2.0
- `androidx.fragment:fragment` - 1.7.0 -> 1.8.1
- `androidx.lifecycle:lifecycle-process` - 2.7.0 -> 2.8.3
- `androidx.lifecycle:lifecycle-viewmodel` - 2.7.0 -> 2.8.3
- `androidx.vectordrawable:vectordrawable-animated` - 1.1.0 -> 1.2.0
- `com.google.auto.value:autovalue-annotations` - 1.10.4 -> 1.11.0
- `com.google.guava:guava` - 32.0.1android -> 33.2.1android
- `org.jetbrains.kotlin:kotlin-stdlib` - 1.9.23 -> 2.0.0
- `org.jetbrains.kotlin:kotlin-stdlib-common` - 1.9.23 -> 2.0.0
- `org.jetbrains.kotlin:kotlin-stdlib-jdk7` - 1.9.23 -> 2.0.0
- `org.jetbrains.kotlin:kotlin-stdlib-jdk8` - 1.9.23 -> 2.0.0
- `org.jetbrains.kotlinx:kotlinx-coroutines-android` - 1.8.0 -> 1.8.1
- `org.jetbrains.kotlinx:kotlinx-coroutines-core` - 1.8.0 -> 1.8.1
- `org.jetbrains.kotlinx:kotlinx-coroutines-playservices` - 1.8.0 -> 1.8.1

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1028,24 +1028,24 @@
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-crashlytics",
-        "version": "19.0.1",
-        "nugetVersion": "119.0.1",
+        "version": "19.0.3",
+        "nugetVersion": "19.0.3",
```
Comment (jpobst) [inline config.json:1032]: 119

File: config.json [hunk 2]
```diff
@@ -1028,24 +1028,24 @@
...
         "nugetId": "Xamarin.Firebase.Crashlytics",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-crashlytics-ktx",
-        "version": "19.0.1",
-        "nugetVersion": "119.0.1",
+        "version": "19.0.3",
+        "nugetVersion": "19.0.3",
```
Comment (jpobst) [inline config.json:1040]: 119

File: config.json [hunk 3]
```diff
@@ -1028,24 +1028,24 @@
...
         "nugetId": "Xamarin.Firebase.Crashlytics.Ktx",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-crashlytics-ndk",
-        "version": "19.0.1",
-        "nugetVersion": "119.0.1",
+        "version": "19.0.3",
+        "nugetVersion": "19.0.3",
```
Comment (jpobst) [inline config.json:1048]: 119

File: config.json (hunk 1 above)
Comment (moljac) [reply config.json:1032]: I think it is update script, but not in all cases. I fixed 10+ of such errors. I will check next time.
Only GPS. AX seems to be OK. OK there is no hundredts prefix.

File: config.json (hunk 2 above)
Comment (moljac) [reply config.json]: fixed

File: config.json (hunk 3 above)
Comment (moljac) [reply config.json]: fixed

**Review by jpobst — APPROVED** [review]

**Pull request merged**